tests/test_box.py

```diff
@@ -39,6 +39,14 @@ def test_grow(box):
     assert bx.num_pts == (134**3)
 
 
+def test_slab(box):
+    """box.make_slab"""
+    box.make_slab(direction=3, slab_index=60)
```

Comment:
```suggestion
    box.make_slab(direction=2, slab_index=60)
```
It looks like ``amrex::makeSlab`` expects 0-indexed directions; should this direction be a 2 rather than a 3?

Reply:
oh, perfect, thank you!